### PR TITLE
[wordpress-docs] Add AI summary feature

### PR DESCRIPTION
**Pull request by KevinBatdorf**
## Description

Adds an action for pro raycast users to view an AI generated summary of the function along with an example snippet.

Includes a shortcut ⌘ ⇧ ↵

Primary action when viewing the summary is to "Continue in Chat"

## Screencast

video is at 2x speed

https://github.com/raycast/extensions/assets/1478421/3213c823-7102-4218-a97d-3cff2214595d

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/wordpress-docs/src/index.tsx [hunk 1]
```diff
@@ -84,8 +84,62 @@ function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
...
+    <>
+      <Action.OpenInBrowser title="Open in Browser" url={searchResult.url} />
+      <Action.CopyToClipboard title="Copy URL to Clipboard" content={searchResult.url} />
+    </>
+  );
+}
+
+function Summary({ searchResult }: { searchResult: SearchResult }) {
+  const item = JSON.stringify(searchResult);
+  const prompt = `Summarize the following from the WordPress docs and give one example of usage in a code block. The context can only be about WordPress. Format the response as if you are providing documentation:\n${item}`;
```
Comment (tillkruss) [inline extensions/wordpress-docs/src/index.tsx:114]: Should we say `from the WordPress documentation`?

Comment (KevinBatdorf) [reply extensions/wordpress-docs/src/index.tsx]: Yeah I updated that now. 

I wonder whether one or the other has an effect on the quality of the output.

**Comment by KevinBatdorf** [comment]
Added a title "AI Generated Summary" to the navigation area otherwise it says the command title ("Search Code Reference"), and this makes it more clear what the source is.

Screen Shot 2023-05-16 at 11 55 08 PM

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**